.github/workflows/conformance-eks.yaml

```
@@ -407,6 +407,7 @@ jobs:
         with:
           role-to-assume: ${{ secrets.AWS_PR_ASSUME_ROLE }}
           aws-region: ${{ matrix.region }}
+          role-duration-seconds: 28800
```

Two things, incidentally: (1) the timeout for the `cleanup` job is 30m, and (2) it might be nice to have a comment after to make it clear how long it is.
```suggestion
          role-duration-seconds: 3600 # 1h
```